main.py

```diff
@@ -1,7 +1,47 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors_fullname = [("George", "Klooney"), ("Kianu", "Reaves"),
+                       ("Scarlett", "Keegan"), ("Will", "Smith"),
+                       ("Jaden", "Smith"), ("Scarlett", "Johansson")]
+
+    for genre in genres:
+        Genre.objects.create(
+            name=genre
+        )
+
+    for fullname in zip(actors_fullname):
```

Comment:
The use of `zip(actors_fullname)` is incorrect here. `zip` is typically used to combine two or more iterables. Since `actors_fullname` is already a list of tuples, you should iterate directly over it: `for fullname in actors_fullname:`.